Infrastructure/Services/AuthorityService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Core.Entities;
+using Core.Interfaces;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Infrastructure.Services
+{
+    public class AuthorityService : CUDService<Authority>, IAuthorityService
+    {
+        public AuthorityService(Donos_Context context) : base(context)
```

Comment:
Jest to co prawda jedyny kontekst, ale warto pisać, że to dbContext (jest bardziej self-explanatory i nie trzeba później zmieniać jak dojdzie kolejny kontekst)